core/src/main/scala/io/finch/internal/package.scala

```diff
@@ -121,4 +122,9 @@ package object internal {
       new String(array, begin, end - begin, cs.name)
     }
   }
+
+  private[internal] val dummieExecutionContext: ExecutionContext = new ExecutionContext {
```

Comment:
I'd probably prefer this being its own file under `internal` package (perhaps as an `object`). Also no need to limit its visibility as it has `internal` in its name.